OpenQA/Isotovideo/CommandHandler.pm

```diff
@@ -119,6 +119,7 @@ sub _send_to_cmd_srv ($self, $data) { myjsonrpc::send_json($self->cmd_srv_fd, $d
 sub _send_to_backend ($self, $data) { myjsonrpc::send_json($self->backend_fd, $data) }
 
 sub send_to_backend_requester ($self, $data) {
+    return undef unless $self->backend_requester;
     myjsonrpc::send_json($self->backend_requester, $data);
     $self->backend_requester(undef);
```

Comment:
this looks like it could be just this

```
myjsonrpc::send_json($self->backend_requester, $data)  unless $self->backend_requester;
$self->backend_requester(undef);
```

Reply:
I guess you mean `myjsonrpc::send_json($self->backend_requester, $data) if $self->backend_requester`. But then we don't need to set it to undef when it's already undef so I think my proposal is cleaner

Reply:
yes the correct syntax is with `if`.
not sure I understatnd the logic. `myjsonrpc::send_json($requester, $data);` should be executed only if `backend_requester` is not _undef_. Correct me but `myjsonrpc::send_json($self->backend_requester, $data) if $self->backend_requester` would return _undef_ if condition is false anyway. likely you wont even need the `$self->backend_requester(undef);`. Even sure I am not sure about that.